src/esp/core/managedContainers/ManagedContainer.h

```diff
@@ -282,7 +290,7 @@ class ManagedContainer : public ManagedContainerBase {
 
   /**
    * @brief Get a reference to, or a copy of, the managed object identified by
-   * the @p managedObjectID, depending on @ref Access value.  This is the
+   * the @p managedObjectID, depending on @p Access value.  This is the
```

Comment:
Doc and API usability tip: have the template parameter accessible also through a `constexpr static` member (or a `typedef`) that you can then `@ref` to, so people have an actual possibility to know what the `Access` is. (Yes, it would be better if we could just link to template parameter docs but Doxygen can do only so much.)

For example the Magnum [Vector type](https://doc.magnum.graphics/magnum/classMagnum_1_1Math_1_1Vector.html#pub-types) has a `size` and `T` parameter, which are then exposed as `Size` and `Type`.